django/contrib/admindocs/views.py

```diff
@@ -411,8 +411,7 @@ def simplify_regex(pattern):
     """
     pattern = replace_named_groups(pattern)
     pattern = replace_unnamed_groups(pattern)
-    # clean up any outstanding regex-y characters.
-    pattern = pattern.replace('^', '').replace('$', '').replace('?', '')
+    pattern = replace_metacharacters(pattern)
```

Comment:
I think we can get rid of `replace_metacharacters()` and just do the `re.sub()` here.